### PR TITLE
fix: move v9 migration packages to root and update storybook setup to fix docs examples

### PR DESCRIPTION
**Pull request by Hotell**
## Previous Behavior

demo source is using invalid imports

- migration packages don't follow v9 storybook scaffolding patterns, thus producing invalid docs code examples that mislead consumers.

## New Behavior

demo source is using valid imports

- updates codesandbox-babel-loader plugin to properly render imports for migration packages
- normalizes storybook scaffolding to match v9 rules (via `yarn nx workspace-generator migrate-converged-pkg --name=@fluentui/react-migration-v0-v9` and import manual change)
- adds simple eslint rule to avoid use to import from `src` by accident

## Related Issue(s)

- Fixes https://github.com/microsoft/fluentui/issues/26961, https://github.com/microsoft/fluentui/issues/26948

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/7bf995b3b64f3ffb07291fc6f7efedb00611f2de'>7bf995b3b64f3ffb07291fc6f7efedb00611f2de</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## 🕵 fluentuiv8 No visual regressions between this PR and main
<div id="vrtCommentfluentuiv8"/>

File: apps/public-docsite-v9/.storybook/main.js [hunk 1]
```diff
@@ -19,8 +19,8 @@ module.exports = /** @type {Omit<import('../../../.storybook/main'), 'typescript
     '../src/**/*.stories.mdx',
```
Comment (fabricteam) [inline apps/public-docsite-v9/.storybook/main.js:19]: ## [🕵 FluentUI-v0 Open the Visual Regressions report to inspect the 1 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=26982&commitId=4fcbb5d153ff86164317162b4bc8c8785bf54980&env=prod)

✅ There was 0 screenshots added, 0 screenshots removed, 1259 screenshots unchanged, 0 screenshots with different dimensions and 1 screenshots with visible difference.


  <details><summary><b>unknown</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Dropdown.Dropdown Example Loading Teams High Contrast.chromium.png|222| Changed|

</details>


<div id="vrtCommentFluentUI-v0"/>

**Review by miroslavstastny — APPROVED** [review]

File: packages/react-components/react-migration-v8-v9/stories/ThemeShim/createV9Theme/Description.md [hunk 2]
```diff
@@ -1 +1,3 @@
 The createV9Theme shim method allows you to create a v9 Theme from a v8 Theme.
+
+hovno
```
Comment (layershifter) [inline packages/react-components/react-migration-v8-v9/stories/ThemeShim/createV9Theme/Description.md:3]:
```suggestion
```

Comment (layershifter) [reply packages/react-components/react-migration-v8-v9/stories/ThemeShim/createV9Theme/Description.md:3]: No thanks 😁

File: packages/react-components/react-migration-v8-v9/stories/ThemeShim/createV9Theme/index.stories.tsx [hunk 3]
```diff
@@ -200,7 +200,7 @@ export const Playground = () => {
       <FluentProvider theme={v9Theme}>
         <ThemePreviewV9 />
       </FluentProvider>
-      <h3>v9 Theme</h3>
+      <h3>v9 Theme z</h3>
```
Comment (layershifter) [inline packages/react-components/react-migration-v8-v9/stories/ThemeShim/createV9Theme/index.stories.tsx:203]:
```suggestion
      <h3>v9 Theme</h3>
```

**Review by layershifter — APPROVED** [review]

**Review by ling1726 — APPROVED** [review]

**Review by micahgodbolt — APPROVED** [review]

**Review by behowell — APPROVED** [review]

**Pull request merged**